src/main/java/seedu/address/model/house/Landed.java

```diff
@@ -0,0 +1,20 @@
+package seedu.address.model.house;
+
+import seedu.address.model.person.Person;
+
+public class Landed extends House {
+
+    /**
+     * Constructs a {@code Level}.
+     *
+     * @param postalCode The postal code of the house.
+     * @param street     The street of the house.
+     */
+    public Landed(UnitNumber unitNumber, PostalCode postalCode, Street street) {
+        super(unitNumber, postalCode, street);
```

Comment:
I think to show proper inheritance we can separate this out to something like:

from:
`super(unitNumber, postalCode, street);` 
to 
`super(postalCode, street);`
`this.unitNumber = unitNumber;`